amgut/lib/data_access/test/test_ag_data_access.py

```diff
@@ -312,139 +318,137 @@ def test_getAnimalParticipantsNotPresent(self):
 
     def test_getParticipantSamples(self):
         i = "d6b0f287-b9d9-40d4-82fd-a8fd3db6c476"
-        res = self.ag_data.getParticipantSamples(i, "REMOVED-0")
-        exp = [{'status': None,
-                'sample_time': datetime.time(11, 55),
-                'notes': 'REMOVED',
-                'barcode': '000028432',
-                'sample_date': datetime.date(2015, 6, 7),
-                'site_sampled': 'Stool'}]
-        self.assertEqual(res, exp)
+        names = self.ag_data.get_participant_names_from_ag_login_id(i)
+        # collect results for ALL names
+        res = [self.ag_data.getParticipantSamples(i, name)
+               for name in names]
+        exp = {'sample_time': datetime.time(11, 55),
+               'barcode': '000028432',
+               'sample_date': datetime.date(2015, 6, 7),
+               'site_sampled': 'Stool'}
+        # transform results
+        res = [dict((k, r[0][k]) for k in exp.keys()) for r in res if r != []]
+        # only look at those fields, that are not subject to scrubbing
+        self.assertIn(exp, res)
 
         i = "d8592c74-9694-2135-e040-8a80115d6401"
-        res = self.ag_data.getParticipantSamples(i, "REMOVED-0")
+        # collect all results for ALL names for this ag_login_id and remove
+        # field "notes" since it gets scrubbed.
+        names = list(set([name for name in
```

Comment:
I don't think you need the call to list, since you can iterate over the elements of a set too. I would rewrite this as:

```python
names = set(ag_data.get_participant_names_from_ag_login_id(i))
```